src/packages/inputnumber/inputnumber.taro.tsx

```
@@ -228,18 +205,9 @@ export const InputNumber: FunctionComponent<
   }
   const handleBlur = (e: React.FocusEvent<HTMLInputElement>) => {
     setFocused(false)
-    onBlur?.(e)
+    onBlur && onBlur(e)
     const valueStr = parseValue(e.target.value)
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)